eth/rpc/backend/tracing.go

```diff
@@ -18,7 +18,10 @@ import (
 
 // TraceTransaction returns the structured logs created during the execution of EVM
 // and returns them as a JSON object.
-func (b *Backend) TraceTransaction(hash gethcommon.Hash, config *evm.TraceConfig) (any, error) {
+func (b *Backend) TraceTransaction(
+	hash gethcommon.Hash,
+	config *evm.TraceConfig,
+) (res json.RawMessage, err error) {
 	// Get transaction by hash
```

Comment:
Since we `json.Marshal` the `any` results, the function signature can have stronger typing